core/lib/zksync_core/src/eth_sender/eth_tx_aggregator.rs

```diff
@@ -448,7 +448,9 @@ impl EthTxAggregator {
                 f.encode_input(&op.into_tokens())
                     .expect("Failed to encode execute transaction data")
             }
-        }
+        };
+
+        (calldata, None)
```

Comment:
so we always return None as secont param? that's surprising, please add commetn explaining

Reply:
Should it be `TxData` enum? With calldata option and eip4844 data?

Reply:
Created a `struct` instead of enum, that should be enough